src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -87,26 +110,32 @@ void DlgPrefReplayGain::slotResetToDefaults() {
 
 void DlgPrefReplayGain::slotSetRGEnabled() {
     if (EnableGain->isChecked()) {
-        config->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(1));
+        m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(1));
     } else {
-        config->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(0));
+        m_pConfig->set(ConfigKey(kConfigKey,"ReplayGainEnabled"), ConfigValue(0));
     }
     slotUpdate();
     slotApply();
 }
 
-void DlgPrefReplayGain::slotSetRGAnalyzerEnabled() {
+void DlgPrefReplayGain::slotSetRGAnalyzerChanged() {
+    bool rg1 = radioButtonRG1->isChecked();
```

Comment:
Suggestion: With two variables bool rgEnabled and int rgVersion the code might be more readable. I would prefer const member functions isReplayGainAnalyserEnabled() and getReplayGainVersion() instead of local variables.